test/experimental/new_embed_test.html

```diff
@@ -14,35 +14,49 @@
 
     <title>DartPad</title>
 
+    <link href='https://fonts.googleapis.com/css?family=Roboto'
+          rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Inconsolata:400,700'
+          rel='stylesheet' type='text/css'>
+
+    <link rel="stylesheet" type="text/css" href="../packages/octicons_css/octicons_4.4.0.css">
+    <link rel="stylesheet" type="text/css" href="../packages/primer_css/primer_11.0.0.css">
+
     <link rel="stylesheet" type="text/css" href="new_embed.css">
 
     <!-- Test package stuff -->
     <link rel="x-dart-test" href="new_embed_test.dart">
     <script src="packages/test/dart.js"></script>
 
-<body>
-<header>
-    <div class="tab" selected id="editor-tab">
-        <a href="#">Editor</a>
-    </div>
-    <div class="tab" id="test-tab">
-        <a href="#">Test</a>
+    <script defer src="../scripts/polymerpatch.js"></script>
+</head>
+
+<body class="d-flex flex-column">
+
+<nav class="UnderlineNav px-2">
+    <div class="UnderlineNav-body">
+        <a id="editor-tab" href="#" class="UnderlineNav-item selected" selected>Your code</a>
+        <a id="test-tab" href="#" class="UnderlineNav-item">Test</a>
+        <a id="console-tab" href="#" class="UnderlineNav-item">Console</a>
```

Comment:
`Output`?

Reply:
Are you suggesting renaming "console" to "output" here, or that we add an HTML output tab similar to the one the playground UI has?

The former is no problem. If it's the latter, I'm not sure how to work that into the design yet (or whether it's a good idea, since I don't see HTML output in the current embedding demo).